src/raftstore/store/worker/split_check.rs

```diff
@@ -86,16 +106,30 @@ impl<'a> MergedIterator<'a> {
     ) -> Result<MergedIterator<'a>> {
         let mut iters = Vec::with_capacity(cfs.len());
         let mut heap = BinaryHeap::with_capacity(cfs.len());
+        let mut write_cf_pos = 0;
         for (pos, cf) in cfs.into_iter().enumerate() {
+            let write_cf = cf == &CF_WRITE;
             let iter_opt =
                 IterOption::new(Some(start_key.to_vec()), Some(end_key.to_vec()), fill_cache);
             let mut iter = db.new_iterator_cf(cf, iter_opt)?;
             if iter.seek(start_key.into()) {
-                heap.push(KeyEntry::new(iter.key().to_vec(), pos, iter.value().len()));
+                heap.push(KeyEntry::new(
+                    iter.key().to_vec(),
+                    pos,
+                    iter.value().len(),
+                    write_cf,
```

Comment:
This is a bit ugly. How about saving the `cfs` in the `MergedIterator` so that we can know the exact cf name of an entry by the `pos`?